mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import cn from "classnames";
+import styles from "./Input.module.scss";
+
+
+interface InputProps {
+  type?: HTMLInputTypeAttribute;
+  placeholder?: string;
+  className?: string | Array<string>;
+  inputMode?: "none" | "text" | "tel" | "url" | "email" | "numeric" | "decimal" | "search" | undefined;
+  disabled?: boolean;
+  required?: boolean;
+}
+
+export const Input: FC<InputProps> = ({
+  type = "text",
+  placeholder = "",
+  className = "",
+  disabled = false,
+  required = false,
+  inputMode,
```

Comment:
let's try to avoid props destructuring currently. I think context should help to read code easily and differentiate props and other variables.